README.md

```diff
@@ -17,7 +17,7 @@ What are constructor functions and the `new` keyword? What is a prototype? Descr
 
 Your Answer:
 ```text
-
+<!-- We use constructor functions to easily create and model different objects that share the same properties. It uses the keyword "this" to create a new object. The 'new' keyword is used to create a new object for that method, and the 'new' keyword calls the constructor function for the new object. We use a prototype when we want the new object to automatically take the properties of the original objects and do not want to rewrite code for the new object (and take up a lot of memory.) -->
 ```
```

Comment:
Couple notes...
* `this` itself doesn't create a new object, but it allows us to attach properties / methods to an instance of an object.
* This one's tricky, but `new` doesn't call the constructor -- `constructorFunction( )` does. However, `new` allows the object created by the constructor to inherit from its prototype.

You're right on the other counts though!